css/master.css

```diff
@@ -1,39 +0,0 @@
-nav li:after {
-    content: '·';
-    width: 13px;
-    height: 13px;
-    display: list-item;
-    margin-top: -40px;
-    float: right;
-    font-size: 20pt;
-    padding-left: 10px;
-}
-nav li:last-child:after {
-    content: '';
-}
-#partners li {
-    list-style: none;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-}
-#partners img {
-    max-width: 100px;
-    max-height: 100px;
-}
-html {
-    height: 100%;
-}
-body {
-    position: relative;
-    min-height: 100%;
-}
-footer.navbar {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    right: 0;
-}
-main {
-    padding-bottom: 250px;
-}
```

Comment:
Split the following into two separate commits:

1. The conversion of the existing CSS into SASS
2. The introduction of new styling (e.g. the styling of `.support`)